source/cps/install.rst

```diff
@@ -0,0 +1,64 @@
+安装
+====
+
+1.  下载:
+
+    首次下载，可以在 http://www.eas.slu.edu/eqc/eqc_cps/CPS/cpslisc.html 填写个人信息，
+    提交源码下载申请。
+
+    以后也可以直接在 http://www.eas.slu.edu/eqc/eqc_cps/Download/ 下载源码，文件名类似
+    ``NP330.DATE.tgz``，其中 DATE 代表最近的更新日期。
```

Comment:
This download link is not available on the CPS website. I don't remember how I find the link, but I think it may not be a good idea to make the "private" link public.